front-end/src/components/analysis-page/AnalysisPage.jsx

```diff
@@ -44,47 +44,47 @@ export class AnalysisPage extends Component {
+                        className="ml-3"
                         onClick={this.openIntentMenu}
                         aria-label="intent menu button"
                         data-testid="intent-menu-button"
                         aria-haspopup="menu"
+                        tabIndex={this.state.showIntentMenu ? "-1" : "0"}
                       >
-                        <label
-                          className="cursor-pointer"
-                        >
+                        <label className="cursor-pointer">
                           <Hamburger />
                         </label>
                       </button>
                     </nav>
                 }
                 {
-                    this.state.showIntentMenu &&
-                    <div
-                        className="justify-start px-7 mr-1 py-3 h-1">
-                    </div>
+                    this.state.showIntentMenu
                 }
-                <div /* first line of className is tooltip styling */
-                  className="relative before:z-10 before:absolute before:left-1/2 before:-bottom-3 before:w-max before:max-w-xs before:-translate-x-1/2 before:translate-y-full before:rounded-lg before:bg-gray-200 before:px-2 before:py-1.5 before:text-off-white before:invisible before:content-[attr(tooltip)] after:z-10 after:absolute after:left-1/2 after:-bottom-3 after:h-0 after:w-0 after:-translate-x-1/2 after:border-8 after:border-b-gray-200 after:border-l-transparent after:border-t-transparent after:border-r-transparent after:invisible hover:before:visible hover:after:visible
-                  cursor-pointer m-3 w-[11em] mx-auto"
-                  role="button"
-                  onClick={this.openMainPage}
+                <div className="justify-self-center">
+                  <button /* first line of className is tooltip styling */
+                  className="relative before:z-10 before:absolute before:left-1/2 before:-bottom-3 before:w-max before:max-w-xs before:-translate-x-1/2 before:translate-y-full before:rounded-lg before:bg-gray-300 before:px-2 before:py-1.5 before:text-off-white before:invisible before:content-[attr(tooltip)] after:z-10 after:absolute after:left-1/2 after:-bottom-3 after:h-0 after:w-0 after:-translate-x-1/2 after:border-8 after:border-b-gray-300 after:border-l-transparent after:border-t-transparent after:border-r-transparent after:invisible hover:before:visible hover:after:visible
+                  m-3 w-[11em] 2xl:w-[15em] mx-auto"
                   aria-label="Home Button"
                   tooltip="Click here to return to homepage"
                   data-testid="home-button"
-                >
-                  <Title className="md:h-full h-20" />
+                  onClick={this.openMainPage}
+                  tabIndex={this.state.showIntentMenu ? "-1" : "0"}
+                  >
+                      <label className="cursor-pointer">
+                        <Title />
```

Comment:
try moving this into the `<Title />` element. It might not work, but I expect it will be slightly cleaner for the user (rn the active state highlights the tooltip too)

Reply:
Title doesn't have a tab element (hence why I put it in the button) so I moved the tooltip into the div above instead 👍

Reply:
epic